### DISCUSSION TITLE
New Proxies every 2 or 3 days I WILL BE POSTING HERE ARE SOME FOR YALL

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1313] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Here are the Proxies for people who have goguardian, I have it too and if these links work for me it should for u. Any problems please tell me in comments. Link 1 https://mhmmm.minecraft.pe/ <----- INTERSTELLAR-STATUES-WORKING Link 2 http://burngraingasesixlschoology.lovethosetrains.com/
http://ixlschoologyschool.lovethosetrains.com/
http://ixlschoology2828.lovethosetrains.com/
http://iloveschoolmath.lovethosetrains.com/
http://burngraingasesixlschoology.lovethosetrains.com/
https://funny-number.69.mu/
THESE DO HAVE ADS BUT IF IT WORKS-DOGE UNBLOCKERS
https://eu.anything-thinking.ipv64.net/
https://anything-thinking.ipv64.net/
https://www.obitrani2.ipv64.net/
https://birutarou.ipv64.net/
https://wwwwww.classroom-google-co.ipv64.net/
https://superapple.ipv64.net/
https://we.adbox.ipv64.net/
https://ea.obitrani.ipv64.net/
https://birutarou.ipv64.net/
https://anything-thinking.ipv64.net/
https://adbox.ipv64.net/
https://iloveorocu.ipv64.net/
https://anything-thinking.ipv64.net/
https://adbox-v7.ipv64.net/
https://adbox-v2.ipv64.net/
https://adbox-v3.ipv64.net/
https://adbox-v4.ipv64.net/
https://adbox-v6.ipv64.net/
this is working
http://mathclassisnotthebest.et-navigator.com/
this worked-TINFOIL
https://www.4digitalsignage.com/content/showtimes/beltway/miniProxy.php
may not work

THANK U FOR YOUR APPRECIATION

**Comment by ghost** [comment]
Yall enjoy

**Comment by Sam-hacks1** [comment]
no way none of these work

**Reply by akabutnicer** [reply]
for you, that is.

**Reply by ghost** [reply]
Does anyone have problems

**Reply by schoolexploitkid** [reply]
ye they send me to some virus link or smth that ublock blocked

**Comment by awsome-webdev** [comment]
Thanks

**Reply by ghost** [reply]
thanks ,  I ton of them work , how do you make them though

**Comment by ghost** [comment]
Dodge unblocker is garbage infested with ads.

**Reply by schoolexploitkid** [reply]
it sends me to malware :(

**Reply by ghost** [reply]
Fr horrible service

**Comment by Sam-hacks1** [comment]
how can I make my own site will that because they don't work

**Comment by Luigi-Master** [comment]
For the people who have securly, Link 1 is the only one that works

**Comment by dudedecliedddddddd** [comment]
all blocked (I have lightspeed)

**Comment by ghost** [comment]
dudedecliedddddddd I am sorry to hear this but I will give you more proxies

**Comment by ghost** [comment]
All blocked I have contentkeeper

**Reply by ghost** [reply]
sry

**Comment by schoolexploitkid** [comment]
@S-PScripts lock this discussion
